File: Data Structures & Algorithms/concatenation-of-array/submission-10.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:

        ans = [0]*2*len(nums)
        ans[0:len(nums)] = nums
        for i  in range(len(nums)):
            ans[1*len(nums) + i] = nums[i]

        return ans
        

File: Data Structures & Algorithms/max-consecutive-ones-iii/submission-10.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number}
     */
    longestOnes(nums, k) {
        let l = 0;
        let count = 0;

        for(let r=0; r< nums.length; r++){
            if(nums[r] === 0){
                k--
            }

            while(k<0){
                if(nums[l] === 0){
                    k++;
                }
                l++;
            }
            count = Math.max(count, r - l + 1);
        }

        return count;
    }
}
